spec/examples/hourly_rule_spec.rb

```diff
@@ -42,10 +42,10 @@ module IceCube
         schedule = Schedule.new(t0 = Time.local(2013, 11, 3, 0, 0, 0))
         schedule.add_recurrence_rule Rule.hourly
         schedule.first(4).should == [
-          Time.local(2013, 11, 3, 0, 0, 0),             # -0700
-          Time.local(2013, 11, 3, 1, 0, 0) - ONE_HOUR,  # -0700
-          Time.local(2013, 11, 3, 1, 0, 0),             # -0800
-          Time.local(2013, 11, 3, 2, 0, 0),             # -0800
+          Time.new(2013, 11, 3, 0, 0, 0, '-07:00'),
```

Comment:
A coworker pointed out to me that I am, indeed, confused - there are two instances of 1:00 AM on Nov 3rd, 2013, and `Time::local` has to pick one. I'd expect it to be the first one, but picking the second one isn't necessarily incorrect behavior. Because of this ambiguity, though, I still think the best solution is to write these tests explicitly, so that Ruby's semi-arbitrary choice of which 1:00 AM to return doesn't affect them.